internal/api/terminal_manager.go

```diff
@@ -136,16 +136,24 @@ func (s *terminalStateManager) startStream(key store.Key, container string) cont
 
 	eventType := event.NewTerminalEventType(key.Namespace, key.Name, container)
 
-	instance, err := terminal.NewTerminalInstance(ctx, s.config.ClusterClient(), logger, key, container, "/bin/sh", s.chanInstance)
-	if err != nil {
-		cancelFn()
-		return cancelFn
-	}
-
-	s.instance = instance
+	commands := []string{"sh", "bash", "powershell", "cmd"}
```

Comment:
This should be it's own PR with a changelog entry.

Reply:
Removed and tracked in #1749